client/my-sites/site-settings/disconnect-site/index.jsx

```diff
@@ -0,0 +1,64 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import DocumentHead from 'components/data/document-head';
+import FormattedHeader from 'components/formatted-header';
+import Main from 'components/main';
+import PaginationFlow from './pagination-flow';
+import { getSelectedSiteId, getSelectedSiteSlug } from 'state/ui/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+
+class DisconnectSite extends Component {
+	componentDidMount() {
+		this.verifySiteIsJetpack();
+	}
+
+	componentDidUpdate() {
+		this.verifySiteIsJetpack();
+	}
+
+	verifySiteIsJetpack() {
+		if ( this.props.siteIsJetpack === false ) {
+			this.redirectToGeneral();
+		}
+	}
+
+	redirectToGeneral = () => {
+		const { siteSlug } = this.props;
+		page( '/settings/general/' + siteSlug );
+	};
+
+	render() {
+		const { translate } = this.props;
+		return (
+			<Main className="disconnect-site why site-settings">
+				<DocumentHead title={ translate( 'Site Settings' ) } />
+				<FormattedHeader
```

Comment:
There are some criteria for which types of sites should see this section, ie is it Jetpack and not Atomic (AT)?

Some places we've started showing placeholders until we have all relevant information to render a component. Do you think it would be a good fit here?

Reply:
@tyxla, would you say we still need to use the ```Placeholder``` since we moved the access checks to a HoC ?

Reply:
@AnnaMag yes. Placeholder will be relevant for the case when the user visits the page with a clean Redux state, and will be shown while the sites are loading, similar to how the other settings main sections work.

Reply:
Ah yes, now I understand why we need it. It is included now.